src/lib/navigation.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Builds a URL for navigating to the game view page with the specified parameters
+ */
+export function buildGameViewUrl(params: {
+  tournamentId: number;
+  groupId: number;
+  round: number;
+  participantId: number;
+}): string {
+  const url = new URL("/partien", window.location.origin);
+  url.searchParams.set("tournamentId", params.tournamentId.toString());
+  url.searchParams.set("groupId", params.groupId.toString());
+  url.searchParams.set("round", params.round.toString());
+  url.searchParams.set("participantId", params.participantId.toString());
+
+  return url.toString();
+}
```

Comment:
Vorschlag:

```suggestion
export function buildGameViewUrl(params: {
  tournamentId: number;
  groupId: number;
  round?: number;
  participantId?: number;
}): string {
  const searchParams = new URLSearchParams();
  searchParams.set("tournamentId", params.tournamentId.toString());
  searchParams.set("groupId", params.groupId.toString());
  searchParams.set("round", params.round.toString());
  searchParams.set("participantId", params.participantId.toString());

  return `/partien?${searchParams.toString()}`
}
```

Das solltest du dann sowohl für die Partien Seite als auch für den Kalender benutzen können

Reply:
done